packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -412,6 +412,9 @@ function transform(root: IRNode, codeGen: CodeGen): t.Expression {
                 if (attr.name === 'id') {
                     return codeGen.genScopedId(attr.value);
                 }
+                if (attr.name === 'spellcheck') {
```

Comment:
this is the only functionality added. the rest is tests.

Reply:
Do we have a separate issue for normalizing expression values?

Reply:
https://github.com/salesforce/lwc-rfcs/pull/21 which is now included in this PR